pkg/daemon/powershell/powershell.go

```diff
@@ -15,7 +15,7 @@ type commandRunner struct{}
 
 // Run runs the command with the PowerShell on PATH
 func (r *commandRunner) Run(cmd string) (string, error) {
-	out, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
+	out, err := exec.Command("powershell", "-executionpolicy", "bypass", "-command", cmd).CombinedOutput()
```

Comment:
should we also set `-NonInteractive`? I think it would help to fail fast if some command results in user input required